apps/cli/src/commands/project/create.ts

```diff
@@ -0,0 +1,139 @@
+import { intro, isCancel, outro, select, spinner, text } from "@clack/prompts";
+import { Args, Command, Flags } from "@oclif/core";
+import axios, { AxiosError, AxiosResponse } from "axios";
+import { blue, bold, cyan, grey, red, underline } from "kleur/colors";
+import { triggerCancel } from "../../lib/helpers";
+import { getCliConfigFromKeyStore } from "../../lib/keyStore";
+
+type NewProjectResponse = {
+  name: string;
+  slug: string;
+  createdAt: Date;
+};
+
+const loader = spinner();
+const API_BASE = process.env.API_BASE || `http://localhost:3000`;
+const LINKS = {
+  projectsPage: `${API_BASE}/projects`,
+  documentation: `${API_BASE}/docs/cli/projects`,
+  auth: `${API_BASE}/cli/auth`,
+};
+
+export default class ProjectCreate extends Command {
+  static description = `Create a new Envless project.\n${grey(
+    `⚡ Please make sure you are running this command from your development project's root directory and make sure you're authenticated. If not, please do so at \n${cyan(
+      LINKS.auth,
+    )}`,
+  )}`;
+  static examples = [
+    `
+      $ envless project create
+      $ envless project create {project name} {project slug}
+      `,
+  ];
+
+  static flags = {
+    name: Flags.string({
+      description: `This is the project's name`,
+      char: "n",
+      dependsOn: ["slug"],
+    }),
+    slug: Flags.string({
```

Comment:
We should not be asking for project slug, instead auto generate on the API, if the project `slug` is taken, please append it with some random integer to make sure its unique.

Reply:
thanks for this. I'd make the updates right away